### PR TITLE
PD 346 Bump deployments

### PR DESCRIPTION
**Pull request by andreipope**
PD 346 Bump deployments

### CODE REVIEW

File: static/samples/k8s/portworx-pxc-grafana.yaml [hunk 1]
```diff
@@ -6046,49 +6046,52 @@ spec:
   selector:
     app: grafana
 ---
-apiVersion: extensions/v1beta1
```
Comment (rodrigc) [inline static/samples/k8s/portworx-pxc-grafana.yaml]: The indentation of this deployment is 2 spaces too far to the right.

File: static/samples/k8s/grafana/prometheus-operator.yaml [hunk 2]
```diff
@@ -71,37 +71,40 @@ metadata:
   name: prometheus-operator
   namespace: kube-system
 ---
-apiVersion: extensions/v1beta1
```
Comment (rodrigc) [inline static/samples/k8s/grafana/prometheus-operator.yaml]: This deployment is indented 2 spaces to the right too far.

File: static/samples/k8s/portworx-pxc-prometheus.yaml [hunk 3]
```diff
@@ -71,40 +71,45 @@ metadata:
   name: prometheus-operator
   namespace: kube-system
 ---
-apiVersion: extensions/v1beta1
```
Comment (rodrigc) [inline static/samples/k8s/portworx-pxc-prometheus.yaml]: This Deployment is indented 2 spaces to the right too far

**Comment by andreipope** [comment]
@rodrigc Hey, Craig, thanks for taking the time to review this. It's updated.

**Pull request merged**